public/dist/css/informationpages.css

```diff
@@ -0,0 +1,46 @@
+.col-md-4 {
+	margin-top: -20px;
+}
+
+.details {
+	color: #0A9BEB;
+	font-weight: bolder;
+	font-family: serif;
+}
+
+.list-group-item:focus {
+	outline-color: #0A9BEB;
+}
+.pagination-style{
```

Comment:
I do not know why the CI is not throwing an error, I'll have to fix that later, but the CSS linter throws these errors:
`public/dist/css/informationpages.css`
 14:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 14:17  ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 17:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 17:20  ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 18:9   ✖  Expected single space after ":" with a single-line declaration                    declaration-colon-space-after   
 20:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 20:5   ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 23:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 23:10  ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 26:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 26:10  ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 29:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 29:14  ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 33:1   ✖  Expected selector ".panel:hover" to come before selector "#event-screen .panel"   no-descending-specificity       
 33:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 33:12  ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 37:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 37:14  ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 40:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 40:11  ✖  Expected single space before "{"                                                  block-opening-brace-space-before
 43:1   ✖  Expected empty line before rule                                                   rule-empty-line-before          
 43:13  ✖  Expected single space before "{"                                                  block-opening-brace-space-before